backend/index.js

```diff
@@ -0,0 +1,26 @@
+const { ApolloServer } = require('apollo-server');
+const { typeDefs } = require('./typeDefs');
+//const { resolvers } = require('./resolvers');
+const { makeAugmentedSchema } = require('neo4j-graphql-js');
+const schema = makeAugmentedSchema ({ typeDefs });
+const neo4j = require('neo4j-driver');
+
+const neoDriver = neo4j.driver(
+  'bolt://localhost:7687',
+  neo4j.auth.basic('neo4j', 'password')
+);
+
+const server = new ApolloServer({ 
+  schema,
+  context: ({ req }) => {
+    const token = req.headers.authorization.replace("Bearer ", "");
+    return {
+      token: token || "",
+      neoDriver
```

Comment:
Here is the reason for:

> GraphQlError Cannot read property 'session' of undefined

You must call it `neo4jDriver` not `neoDriver`. When `neo4j-graphql-js` tries to do something like:
```js
const session = context.neo4jDriver.session()
```
 then `context.neo4jDriver` is `undefined`.